packages/cli/src/fingerprint/canonicalize/packageDependencies.js

```diff
@@ -9,7 +9,7 @@ function packageOf(codeObject) {
   }
 
   const ancestors = [codeObject, ...codeObject.ancestors()];
-  let packageObject = ancestors.find((a) => a.type === CodeObject.PACKAGE);
+  let packageObject = ancestors.find((a) => a.type === CodeObjectType.PACKAGE);
```

Comment:
This is a bug that resulted in the package object being missed or mis-identified, resulting in the package dependencies list in the index file being empty.